.github/actions/integration-tests/action.yaml

```diff
@@ -28,15 +25,16 @@ runs:
       run: |
         git fetch origin pull/${{ github.event.number }}/head:PR-${{ github.event.number }}
         git checkout PR-${{ github.event.number }}
-    - name: Set up environment
-      shell: bash
-      run: |
-        sudo echo "127.0.0.1 local.kyma.dev" | sudo tee -a /etc/hosts
-        wget -q -O - https://raw.githubusercontent.com/k3d-io/k3d/main/install.sh | sudo bash
-        [ -n "${KYMA_CLI_UNSTABLE}" ] && sudo curl -Lo /usr/bin/kyma https://storage.googleapis.com/kyma-cli-unstable/kyma-linux
-        [ -z "${KYMA_CLI_UNSTABLE}" ] && sudo curl -Lo kyma.tar.gz "https://github.com/kyma-project/cli/releases/latest/download/kyma_linux_x86_64.tar.gz" && sudo tar -C /usr/bin -zxvf kyma.tar.gz kyma
-        sudo chmod +x /usr/bin/kyma
-    - name: Provision Kyma and run tests
+    - name: Create Single Cluster
+      uses: AbsaOSS/k3d-action@4e8b3239042be1dc0aed6c5eb80c13b18200fc79 #v2.4.0
+      with:
+        cluster-name: "test-cluster-1"
```

Comment:
Why not `kyma-cluster` like it was before or maybe just `k3d-cluster`

Reply:
I guess there is no reason for this name, taken straight from the other action